Tests/UnitTests/Android/app/build.gradle

```diff
@@ -70,25 +70,37 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
 }
 
-task copyScripts {
-    copy
-    {
-        from "../../../node_modules/chai"
-        include "chai.js"
-        into 'src/main/assets/Scripts'
+task copyScripts{
+    doLast {
+        // run copy at execution phase because npm command is not done at configuration phase
+        copy
+        {
+            from "../../../node_modules/chai"
+            include "chai.js"
+            into 'src/main/assets/Scripts'
+        }
+        copy
+        {
+            from "../../../node_modules/mocha"
+            include "mocha.js"
+            into 'src/main/assets/Scripts'
+        }
+        copy
+        {
+            from '../../Scripts'
+            include "*.js"
+            into 'src/main/assets/Scripts'
+        }
```

Comment:
Are we running `npm install` in the source folder? Ideally, we should use the CMake binary folder.

Reply:
Let's talk offline.

Reply:
Not doing it in this PR. Discussion captured here : https://github.com/BabylonJS/JsRuntimeHost/issues/36